mteb/models/model_implementations/slm_models.py

```diff
@@ -0,0 +1,449 @@
+            model_name,
+            torch_dtype=torch.bfloat16,
+            revision=revision,
+            **kwargs,
+        )
+
+        self.processor = ColLFM2Processor.from_pretrained(
+            model_name,
+            revision=revision,
+        )
+        
+        logger.info(f"SLM-ColLFM2 loaded: dim={self.mdl.dim}, device={self.device}")
+
+
+# =============================================================================
+# ColMinistral3 Wrapper
+# =============================================================================
+
+class SLMColMinistral3Wrapper(SLMBaseWrapper):
+    """Wrapper for SLM-ColMinistral3 models (Ministral3 backbone)."""
+
+    def _load_model_and_processor(self, model_name, revision, use_flash_attn, **kwargs):
+        from sauerkrautlm_colpali.models.ministral3.colministral3 import ColMinistral3, ColMinistral3Processor
+
+        self.mdl = ColMinistral3.from_pretrained(
+            model_name,
+            torch_dtype=torch.bfloat16,
+        )
+
+        self.processor = ColMinistral3Processor.from_pretrained(model_name)
+        
+        logger.info(f"SLM-ColMinistral3 loaded: dim={self.mdl.dim}, device={self.device}")
+
+
+# =============================================================================
+# Citations
+# =============================================================================
+
+SAUERKRAUTLM_CITATION = """
+@misc{sauerkrautlm-colpali-2025,
+  title={SauerkrautLM-ColPali: Multi-Vector Vision Retrieval Models},
+  author={David Golchinfar},
+  organization={VAGO Solutions},
+  year={2025},
+  url={https://github.com/VAGOsolutions/sauerkrautlm-colpali}
+}
+"""
+
+COLPALI_CITATION = """
+@misc{faysse2024colpali,
+  title={ColPali: Efficient Document Retrieval with Vision Language Models},
+  author={Faysse, Manuel and Sibille, Hugues and Wu, Tony and Omrani, Bilel and Viaud, Gautier and Hudelot, C\\'eline and Colombo, Pierre},
+  year={2024},
+  eprint={2407.01449},
+  archivePrefix={arXiv},
+  primaryClass={cs.IR}
+}
+"""
+
+
+# =============================================================================
+# ColQwen3 Model Metadata
+# =============================================================================
+
+# ColQwen3-1.7B Turbo: ~1.7B params → 3.4 GB VRAM in bfloat16
+slm_colqwen3_1_7b_turbo = ModelMeta(
+    loader=SLMColQwen3Wrapper,
+    name="VAGOsolutions/SauerkrautLM-ColQwen3-1.7b-Turbo-v0.1",
+    languages=SUPPORTED_LANGUAGES,
+    revision="main",
+    release_date="2025-01-01",
+    modalities=["image", "text"],
+    n_parameters=1_700_000_000,
+    memory_usage_mb=3400,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
+    reference="https://huggingface.co/VAGOsolutions/SauerkrautLM-ColQwen3-1.7b-Turbo-v0.1",
+    similarity_fn_name=ScoringFunction.MAX_SIM,
+    use_instructions=True,
+    training_datasets=None,
```

Comment:
Can you add training datasets to all models?
```suggestion
    training_datasets={"MMarcoReranking", "VDRMultilingualRetrieval"} | COLPALI_TRAINING_DATA,
```